### PR TITLE
upgrade framework to v0.9.0

### PR DESCRIPTION
**Pull request by haoqing0110**
Ref: https://github.com/open-cluster-management-io/ocm/issues/355

### CODE REVIEW

**Comment by qiujian16** [comment]
need some more ut, the gate fails

File: pkg/klusterlet/clusterinfo/controller.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ func (r *ClusterInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	err = r.Client.Status().Update(ctx, newClusterInfo)
 	if err != nil {
-		klog.Error("Failed to update clusterInfo status. error %v", err)
+		klog.Error("Failed to update clusterInfo status. error %w", err)
```
Comment (haoqing0110) [inline pkg/klusterlet/clusterinfo/controller.go:120]: The "SonarCloud Code Analysis" points to this line as 0 coverage on new code, can we just force merge this PR?

Comment (haoqing0110) [reply pkg/klusterlet/clusterinfo/controller.go:120]: @qiujian16 ^^

**Pull request merged**